cmd/swag/main.go

```diff
@@ -30,9 +31,15 @@ const (
 	instanceNameFlag     = "instanceName"
 	overridesFileFlag    = "overridesFile"
 	parseGoListFlag      = "parseGoList"
+	quietFlag            = "quiet"
 )
 
 var initFlags = []cli.Flag{
+	&cli.BoolFlag{
+		Name:    quietFlag,
+		Aliases: []string{q},
```

Comment:
You forgot to put the quotes :)
Error: vet: cmd/swag/main.go:40:21: undeclared name: q